math/mathmore/src/GSLMultiRootFinder.cxx

```diff
@@ -158,7 +158,7 @@ const double * GSLMultiRootFinder::FVal() const {
 }
 const char * GSLMultiRootFinder::Name() const {
    // get GSL name
-   return (fSolver != 0) ? fSolver->Name().c_str() : "";
+   return (fSolver != 0) ? fSolver->Name().c_str() : nullptr;
```

Comment:
The problem of fSolver->Name()->c_str() should have been fixed by my commit of yesterday. 
Strange the compiler is still complaining. Which version ?